guides/documentation_guide.md

```diff
@@ -0,0 +1,28 @@
+# Writing notes for each algorithm
+When creating the notes for algorithms of various categories, please use this template:
+
+```
```

Comment:
Write `Markdown` after the three ticks so proper syntax highlighting is visible.